Source/MediaInfo/MediaInfo_Config_MediaInfo.cpp

```diff
@@ -1788,8 +1789,7 @@ Ztring MediaInfo_Config_MediaInfo::File_Duplicate_Set (const Ztring &Value_In)
     }
     if (!ToReturn.empty())
         ToReturn.erase(ToReturn.begin()); //Remove first ";"
-
-    CS.Leave();
+    }

```

Comment:
I am not a big fan of adding {}, but also not against it if you prefer it + fix compilation issues.